src/main/tv/codely/scala_http_api/video/domain/Video.scala

```diff
@@ -1,14 +1,17 @@
 package tv.codely.scala_http_api.video.domain
 
+import tv.codely.scala_http_api.course.domain.Course
+
 import scala.concurrent.duration.Duration
 
 object Video {
-  def apply(id: String, title: String, duration: Duration, category: String): Video = Video(
+  def apply(id: String, title: String, duration: Duration, category: String, course: Course): Video = Video(
```

Comment:
As previously discussed, we should depend only on the `CourseId` class instead of the whole `Course` one.